017-strimzi-ui-mono-repo.md

```diff
@@ -0,0 +1,53 @@
+
+# Converting Strimzi UI Mono Repo
```

Comment:
I think the name is the most confusing part of this. It is not proposing joining any repos. It is proposing to make the code more modular by splitting it into subproject. I think calling it _Modularize Strimzi UI_ or something would be more understandable and better represent the proposal content.

Reply:
Will do thanks.